setup.cfg

```diff
@@ -1,5 +1,65 @@
+[metadata]
+name = ggshield
+version = 1.18.1
```

Comment:
This duplicates the version number from `ggshield/__init__.py`. Is it possible to read the version from the file instead? If not then you modify `scripts/release` so that the `update_version()` function updates the version in this file too.